<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>blackJack</class>
 <widget class="QDialog" name="blackJack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 100, 0);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 220);
font: 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Opciones</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlayer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(220, 220, 220);
font: italic 13pt &quot;Times New Roman&quot;;

</string>
   </property>
   <property name="text">
    <string>Jugador</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_house">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(220, 220, 220);
font: italic 13pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string>Casa</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCartaJ1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 255, 255);
color: rgb(0, 0, 0);
font: 16pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCartaJ2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 255, 255);
color: rgb(255, 0, 0);
font: 16pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCartaJ3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 255, 255);
color: rgb(0, 0, 0);
font: 16pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCartaM3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 255, 255);
color: rgb(255, 0, 0);
font: 16pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCartaM2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 255, 255);
color: rgb(0, 0, 0);
font: 16pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelCartaM1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 255, 255);
color: rgb(255, 0, 0);
font: 16pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>190</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_pedirCarta">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(220, 220, 220);
font: 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Pedir carta</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_detenerse">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(220, 220, 220);
font: 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>detenerse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelCartaJ1_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 255, 255);
color: rgb(0, 0, 0);
font: 16pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCartaJ2_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 255, 255);
color: rgb(255, 0, 0);
font: 16pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCartaJ3_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 255, 255);
color: rgb(0, 0, 0);
font: 16pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCartaM1_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>420</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 255, 255);
color: rgb(255, 0, 0);
font: 16pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCartaM2_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>420</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 255, 255);
color: rgb(0, 0, 0);
font: 16pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelCartaM3_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>420</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 255, 255);
color: rgb(255, 0, 0);
font: 16pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>21</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_house_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(220, 220, 220);
font: italic 13pt &quot;Times New Roman&quot;;
</string>
      </property>
      <property name="text">
       <string>Jugador:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ptsJug">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;Times New Roman&quot;;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>20</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_house_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(220, 220, 220);
font: italic 13pt &quot;Times New Roman&quot;;
</string>
      </property>
      <property name="text">
       <string>Casa:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ptsMaquina">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;Times New Roman&quot;;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
